From 7d7ed78681797d2dffa1070060e063a508a18ecf Mon Sep 17 00:00:00 2001
From: Jeffrey Gelens <jeffrey@qonvoy.io>
Date: Mon, 30 Oct 2023 19:28:55 +0100
Subject: [PATCH 014/188] Add MacPorts as installation option for MacOS (#8663)

* Added MacPorts as installation option for MacOS

* Added macports to ToC
---
 book/src/install.md | 7 +++++++
 1 file changed, 7 insertions(+)

--- a/book/src/install.md
+++ b/book/src/install.md
@@ -13,6 +13,7 @@
   - [AppImage](#appimage)
 - [macOS](#macos)
   - [Homebrew Core](#homebrew-core)
+  - [MacPorts](#macports)
 - [Windows](#windows)
   - [Winget](#winget)
   - [Scoop](#scoop)
@@ -133,6 +134,12 @@ chmod +x helix-*.AppImage # change permi
 brew install helix
 ```
 
+### MacPorts
+
+```sh
+port install helix
+```
+
 ## Windows
 
 Install on Windows using [Winget](https://learn.microsoft.com/en-us/windows/package-manager/winget/), [Scoop](https://scoop.sh/), [Chocolatey](https://chocolatey.org/)
